python/ray/dashboard/modules/metrics/grafana_dashboard_factory.py

```diff
@@ -30,6 +30,14 @@
     "RAY_GRAFANA_{name}_DASHBOARD_GLOBAL_FILTERS"
 )
 
+# Grafana dashboard layout constants
+# Dashboard uses a 24-column grid with 2-column panels
+PANEL_HEIGHT = 8  # Height of each panel
+PANEL_WIDTH = 12  # Half of dashboard width
```

Comment:
should we calculate this based on `PANELS_PER_ROW` and the full width of a Grafana Dashboard (24)?